client/styles/components/_preferences.scss

```diff
@@ -151,3 +219,37 @@
 .preference__option.preference__canvas:not(:last-child) {
 	padding-right: #{14 / $base-font-size}rem;
 }
+
+/*.tooltip {
```

Comment:
I'm assuming you're still working on the tooltip styles. You don't need to include any prefixing, like `webkit-border-radius` or `moz-border-radius` since webpack uses autoprefixer.

Reply:
Yeah, I will delete them.